server/server.js

```diff
@@ -108,7 +108,7 @@ function handleRequest(request, response) {
   if (parts[0] === "") parts.shift();
 
   var fileExt = path.extname(parsedUrl.pathname);
-  var staticResources = ['.js','.css', '.svg', '.ico', '.png', '.jpg', 'gif'];
+  var staticResources = ['.js','.css', '.svg', '.ico', '.png', '.jpg', '.gif', '.ttf', '.eot', '.woff', '.woff2'];
```

Comment:
we don't have external fonts anymore

Reply:
That`s right. But in my case, I import the fonts with an own Dockerfile and replace the CSS and want to serve the Fonts via the server. So if we can leaf it with this extensions, I would be very happy.